cmd/commandfuncs.go

```diff
@@ -630,7 +630,7 @@ func AdminAPIRequest(adminAddr, method, uri string, headers http.Header, body io
 	if err != nil {
 		return nil, fmt.Errorf("making request: %v", err)
 	}
-	if parsedAddr.IsUnixNetwork() {
+	if parsedAddr.IsUnixNetwork() { //nolint:revive
```

Comment:
What happens otherwise?

Reply:
the revive linter prompts us to remove the empty code block that is currently just comments.